### DISCUSSION TITLE
Question about set team color of {_team} to blue

### DISCUSSION THREAD

**Discussion by Thomas7191** [ShaneBeee/SkBee#658] [General]
I have a question regarding setting the team color with the command set team color of {_team} to blue. 
Is it possible to use a hex color code instead, like: set team color of {_team} to "<##ff455f>"

**Comment by ShaneBeee** [comment]
no its not

**Comment by Thomas7191** [comment]
Hello, Will you add something like this, or not?

**Comment by ShaneBeee** [comment]
no, Minecraft doesn't support custom colours for teams

**Comment by Thomas7191** [comment]
Okay, thank you for the information.
